components/aws/sagemaker/common/_utils.py

```diff
@@ -471,10 +471,52 @@ def create_model_request(args):
 
     return request
 
+def endpoint_name_exists(client, endpoint_name):
+  try:
+      endpoint_name = client.describe_endpoint(EndpointName=endpoint_name)['EndpointName']
+      logging.info("Endpoint exists: " + endpoint_name)
+      return True
+  except ClientError as e:
+      logging.info("Endpoint does not exist")
+  return False
+
+def endpoint_config_name_exists(client, endpoint_config_name):
+  try:
+      config_name = client.describe_endpoint_config(EndpointConfigName=endpoint_config_name)['EndpointConfigName']
+      logging.info("Endpoint Config exists: " + config_name)
+      return True
+  except ClientError as e:
+      logging.info("Endpoint Config does not exist:" + endpoint_config_name)
+  return False
+
+def update_deployed_model(client, args):
+    endpoint_config_name = create_endpoint_config(client, args)
+    endpoint_name = update_endpoint(client, args['region'], args['endpoint_name'], endpoint_config_name)
```

Comment:
same question as before - tags are not being passed as parameter, should we mention this in documentation or maybe just delete and recreate tags?

Reply:
I updated the document to say tags will not be updated